metrics/testutils/testutils.go

```diff
@@ -20,6 +20,7 @@ import (
 	"github.com/stretchr/testify/assert"
 
 	"github.com/uber/jaeger-lib/metrics"
+	"github.com/uber/jaeger-lib/metrics/testutils/metricstest"
```

Comment:
so I was suggesting to move these functions to `metricstest`, since they depend on that type, and therefore moving them is no more a breaking change than moving the factory itself.

Reply:
I moved the former testutils functions into metrics/testutils/metricstest/metricstest.go. I kept the intermediate "testutils" directory because there might be other test packages under that directory in the future, but I don't feel strongly about this.